spec/helpers/applications_helper_spec.rb

```diff
@@ -85,6 +85,12 @@
     end
   end
 
+  describe "decode" do
+    it "should decode the html entities such as &amp; to &" do
```

Comment:
Nice :smile: that should be all good then. I guess Rails is still automatically escaping it.